lib/util.js

```diff
@@ -8,7 +8,7 @@ exports.getLocation = function (message) {
     column: message.column,
   };
 
-  var messageInput = get(messageNode, 'source.input');
+  var messageInput = delve(messageNode, 'source.input');
```

Comment:
Can we remove any helper here? Seems like we can do it manually.

Reply:
Sure, I made the change :)